### PR TITLE
Add dish model and display on /dishes page

### PR DESCRIPTION
**Pull request by aniansson**
Waffle Story: https://waffle.io/CraftAcademy/slow_food_sinatra_nov/cards/583ed6b3c23c072300c77fbe

Changes proposed in this pull request:

Since first pull request, we have created a view to able to see dishes with price. As well as making all our tests go green. 

What I have learned working on this feature: 
- How to write loops in the view file to mix with HTML
- How to run tests in cucumber
- How to write features and steps definitions
- How to use another local server - Shotgun
- How to make routes
- How to make models for the database
- We have been able to follow how the database content finds its way to the view through the @dishes variable

Screenshots: 
pasted_image_at_2016_12_01_12_28

Ready for review!

### CODE REVIEW

File: capybara-201611301443567584050390.html [hunk 1]
```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
```
Comment (AmberWilkie) [inline capybara-201611301443567584050390.html]: Delete these capybara-23402734 files. They are the saved screenshots from when you did `save_and_open_page`. They should not be part of your final code.

File: features/see_dishes.feature [hunk 2]
```diff
@@ -0,0 +1,12 @@
+Feature: As a visitor
+  In order to choose dishes
+  I need to be able to see a menu with dishes
+
+  Scenario: Visitor see dishes
+    When I visit the dishes page
+    # Then I should see the page
```
Comment (AmberWilkie) [inline features/see_dishes.feature]: Delete this comment.

File: features/see_dishes.feature [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+    Then I see "Main Courses"
+    And I see "Meatballs"
+    And I see the price of "$8.99"
+    And I see "Add to" button
```
Comment (AmberWilkie) [inline features/see_dishes.feature]: Why is this button "Add to"? Shouldn't it be something like "Add to Cart" or "Add to Order"?

File: features/step_definitions/see_dishes_steps.rb [hunk 4]
```diff
@@ -0,0 +1,19 @@
+When(/^I visit the dishes page$/) do
+  visit '/dishes'
+end
+
+Then(/^I should see the page$/) do
+  save_and_open_page
+end
+
+Then(/^I see "([^"]*)"$/) do |arg1|
```
Comment (AmberWilkie) [inline features/step_definitions/see_dishes_steps.rb]: Replace `arg1` with something like `content` or `text`. Try not to leave `arg1` in your step defs - replace it with a descriptive variable name so your colleagues can understand your tests.

File: features/step_definitions/see_dishes_steps.rb [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+  expect(page).to have_content arg1
+end
+
+Then(/^I see the price of "([^"]*)"$/) do |arg1|
```
Comment (AmberWilkie) [inline features/step_definitions/see_dishes_steps.rb]: Same.

File: features/step_definitions/see_dishes_steps.rb [hunk 6]
```diff
@@ -0,0 +1,19 @@
...
+  pending # Write code here that turns the phrase above into concrete actions
+end
+
+Then(/^I see "([^"]*)" button$/) do |arg1|
```
Comment (AmberWilkie) [inline features/step_definitions/see_dishes_steps.rb]: Same

File: features/step_definitions/see_dishes_steps.rb (hunk 5 above)
Comment (AmberWilkie) [reply features/step_definitions/see_dishes_steps.rb]: Actually, you should delete these `pending` steps. You aren't using them, so don't leave them in your final code.

File: features/see_dishes.feature [hunk 7]
```diff
@@ -0,0 +1,15 @@
+Feature: As a visitor
+  In order to choose dishes
+  I need to be able to see a menu with dishes
+
+  Background:
+    Given I have a dish called "Meatballs" with price "8.99"
+
+  Scenario: Visitor see dishes
+    When I visit the dishes page
+    # Then I should see the page
```
Comment (AmberWilkie) [inline features/see_dishes.feature]: Delete comments.

File: lib/views/dishes.erb [hunk 8]
```diff
@@ -0,0 +1,12 @@
...
+<% @dishes.each do |dish| %>
+<form class="form" action="index.html" method="post">
+<%= dish.name %>: $<%= dish.price %>
+  <input type="button" name="addto" id="addto" value="Add to" />
+<br />
+<% end %>
+</form>
+
+<form class="form" action="index.html" method="post">
+  <input type="button" name="addto" id="addto" value="CHECKOUT" />
```
Comment (AmberWilkie) [inline lib/views/dishes.erb]: You are going to run into problems later if you make both the "Add to" button and the "CHECKOUT" button have the same id.

File: spec/see_dishes_spec.rb [hunk 9]
```diff
@@ -0,0 +1,20 @@
...
+describe Dish do
+  before do
+    subject = Dish.new
+    subject.name = 'Meatballs'
+    subject.price = 8.99
+    subject.save
+  end
+
+  it "is expected to have a name" do
+    expect(subject.first.name).to eq 'Meatballs'
```
Comment (AmberWilkie) [inline spec/see_dishes_spec.rb]: This used to be `Dish.first.name`. Did that not work? With that code, we are testing that a record was saved to the database.

**Review by AmberWilkie — COMMENTED** [review]
Great work, guys. A few things to look at but overall this should be working code.

File: capybara-201611301443567584050390.html (hunk 1 above)
Comment (aniansson) [reply capybara-201611301443567584050390.html]: Done

File: features/see_dishes.feature (hunk 2 above)
Comment (aniansson) [reply features/see_dishes.feature]: Done

File: features/see_dishes.feature [hunk 10]
```diff
@@ -0,0 +1,14 @@
+Feature: As a visitor
+  In order to choose dishes
+  I need to be able to see a menu with dishes
+
+  Background:
+    Given I have a dish called "Meatballs" with price "8.99"
```
Comment (AmberWilkie) [inline features/see_dishes.feature:6]: What happened to the test for the second dish?

Comment (AmberWilkie) [reply features/see_dishes.feature:6]: Make sure you are also testing that you see it below.

**Review by AmberWilkie — COMMENTED** [review]
Ok good. The test I'm requesting should pass so this should take just a minute.

**Pull request merged**